modules/ROOT/pages/http/http-connector.adoc

```diff
@@ -3,4 +3,155 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Release Notes: xref:release-notes::connector/connector-http.adoc[HTTP Connector Release Notes]
\ No newline at end of file
+Release Notes: xref:release-notes::connector/connector-http.adoc[HTTP Connector Release Notes]
+
+The HTTP connector allows declaring HTTP servers that can listen to
+requests and trigger flows, as well as HTTP clients that can communicate with any
+HTTP service. This means that the HTTP connector effectively allows you to both
+expose and consume HTTP based APIs. 
+
+Examples of HTTP based APIs are that the HTTP connector powers both APIkit
+and RESTConnect.
+
+The HTTP connector also supports HTTPS (TLS) connectivity, provides easy ways to serve static
+resources and secure servers, as well as handling a number of client authentication schemes.
+
+== HTTP Servers
+
+HTTP servers expose services for users or applications to consume. They
+receive data in the form of HTTP requests and return a response. For the connector,
+this means triggering flows when a request is received and answering back once the
+flow finishes. This is all handled by the HTTP Listener, a source the connector provides.
+
+=== Set Up an HTTP Server
+
+Declare an HTTP server by first defining an HTTP Listener connection, which defines the
+host and port where HTTP requests are received.
+
+For example:
+
+[source,xml,linenums]
+----
+<http:listener-config name="serverConfig">
+  <http:listener-connection host="localhost" port="8081"/>
+</http:listener-config>
+----
+
+In this example, the HTTP Listener is named `serverConfig` and listens on a computer's localhost address
+(IP address 127.0.0.1) and on port 8081. After you set up a server in Anypoint Studio or Design Center, 
+you can access this listener at the `+http://localhost:8081+` URL and the server responds depending on 
+how you configure the application.
+
+You can choose specific resource paths to execute a flow, so that when a request
+is received matching that path, Mule runs your flow.
+
+The following example displays either Hello World or Goodbye World depending on how you configure the HTTP connector:
+
+[source,xml,linenums]
+----
+<flow name="hello">
+    <http:listener path="greet" config-ref="serverConfig"/>
+    <set-payload value="Hello World!"/>
```

Comment:
@george-hoffer, why have you modified the examples? This makes the latter ones nonsense.

Reply:
The example that had "excuse" is not a good word choice as it has multiple meanings. "exit" is a clearer action word.